### PR TITLE
Update the issue template with additional information

### PR DESCRIPTION
**Pull request by suricactus**
I have updated the issue template to match the challanges of modern times, such as mobile OS name, QFieldCloud additional info etc.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 22a22ccdbbf7fd4e08a9372df8d1c63497110cd3:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-682b45befee5829d02cfd7dcd71fd97477cbadbc-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-682b45befee5829d02cfd7dcd71fd97477cbadbc-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-682b45befee5829d02cfd7dcd71fd97477cbadbc-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-682b45befee5829d02cfd7dcd71fd97477cbadbc-x86-android.apk)

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,29 +1,38 @@
-[Short description of problem here]
+### Describe the bug
```
Comment (nirvn) [inline .github/ISSUE_TEMPLATE.md:1]:
```suggestion
### Describe the issue
```

**Review by nirvn — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (suricactus) [reply .github/ISSUE_TEMPLATE.md]: Sounds better

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 45bc851d1b4bf549dce877ffd143fcd9542a6925:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-b01053ae7b21bdf556ecfdd7358bc1c760137c7d-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-b01053ae7b21bdf556ecfdd7358bc1c760137c7d-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-b01053ae7b21bdf556ecfdd7358bc1c760137c7d-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-b01053ae7b21bdf556ecfdd7358bc1c760137c7d-x86-android.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for a93d480113000a666cbd895b025f877fbe4b76e7:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-30096062ef1c28ae481dd2a0b0e9d1257591d66e-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-30096062ef1c28ae481dd2a0b0e9d1257591d66e-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-30096062ef1c28ae481dd2a0b0e9d1257591d66e-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-30096062ef1c28ae481dd2a0b0e9d1257591d66e-x86-android.apk)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
 
-### Expected behavior:
-
-[Describe expected behavior here]
-
-### Observed behavior:
-
-[Describe observed behavior here]
+### Observed behavior
+[A clear and concise description of what you actually happened.]
```
Comment (m-kuhn) [inline .github/ISSUE_TEMPLATE.md:14]:
```suggestion
[A clear and concise description of what actually happened.]
```

**Review by marioba — APPROVED** [review]

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 7314bfa69e1a1bb371ae01f43fc7a7be21c9a880:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-d34cf76546541dbdb642943969e046c94a53c914-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-d34cf76546541dbdb642943969e046c94a53c914-arm-android.apk), [arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-d34cf76546541dbdb642943969e046c94a53c914-arm-neon-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-d34cf76546541dbdb642943969e046c94a53c914-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2593-d34cf76546541dbdb642943969e046c94a53c914-x86-android.apk)

**Pull request merged**